TACC/Core-Portal
# FP-387: UI Fix: Job Name vs ID

## Overview: ##

Rename text from "Job ID" to "Job Name" (to match Design and data).

See [request on PR for FP-387](https://github.com/TACC/Frontera-Portal/pull/29#pullrequestreview-420205889).

## PR Status: ##

* Work in Progress.
* Hold.

## Related Jira tickets: ##

* [FP-387](https://jira.tacc.utexas.edu/browse/FP-387)

## Summary of Changes: ##

- Change text

## Testing Steps: ##
1. Load Dashboard.
2. Ensure "Job ID" is not a column header.
3. Ensure "Job Name" is the first column header of "My Recent Jobs" table.

## Notes: ##
